index.js

```diff
@@ -146,6 +146,7 @@ const runChecks = async (context, checkEvent) => {
               periodicCheckModule.ensureAllPullRequestsAreAssigned(context),
               periodicCheckModule.ensureAllIssuesHaveProjects(context),
               staleBuildModule.checkAndTagPRsWithOldBuilds(context),
+              checkPullRequestReviewModule.handleLabels(context)

```

Comment:
This is not required and should not be a periodic check.

Reply:
Was there for testing purposes. Will remove that Sir.